.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
Last time I tested this I found that the `https://github.com/scikit-learn/scikit-learn/issues/new` does not exist, so the "Blank issue" on the bottom of this file does not work.

On your fork, does the link work now? (It should be `https://github.com/lesteve/scikit-learn/issues/new` on a fork)

Reply:
I push this PR branch in main fork `main` branch and https://github.com/lesteve/scikit-learn/issues/new works fine. I also double check that I can go to my fork [issues](https://github.com/lesteve/scikit-learn/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen) click the "New issue" and select the blank template.

Reply:
Maybe there is something subtle I am missing though because even the link with scikit-learn repo https://github.com/scikit-learn/scikit-learn/issues/new works for me ...

Reply:
In the past, https://github.com/scikit-learn/scikit-learn/issues/new exists on main because `blank_issues_enabled: true`.

In any case, it looks like this works. Lets merge and see if there is an issue here.

Reply:
Merged and checked that the link still works. 🥳